pkg/security/secl/compiler/generators/accessors/accessors.go

```diff
@@ -464,14 +464,15 @@ func handleSpecRecursive(module *common.Module, astFiles *AstFiles, spec interfa
 	for _, field := range structType.Fields.List {
 		fieldCommentText := field.Comment.Text()
 		fieldIterator := iterator
+		fieldEvent := event
 
 		var tag reflect.StructTag
 		if field.Tag != nil {
 			tag = reflect.StructTag(field.Tag.Value[1 : len(field.Tag.Value)-1])
 		}
 
 		if e, ok := tag.Lookup("event"); ok {
-			event = e
```

Comment:
basically here we are changing the `event` variable, so this might leak to the next fields